test_wrapper.py

```diff
@@ -256,3 +256,8 @@ def test_load_lazy_nested(self):
         self.assertIsNone(res['a'])
         self.assertIsNone(res['test1']['b'])
         self.assertIsNone(res['test2']['test3']['c'])
+
+    def test_write_unicode(self):
+        l = ['a', 'b']
+        l2 = [unicode(n) for n in l]
+        h5w.add_to_h5(fn, {'l': l2})
```

Comment:
could you also add a scalar unicode value to the test to make sure that both additions below work as expected?